src/components/ResetButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from 'next/router';
+import { useDeploymentPageContext } from '../pages/deployment/DeploymentPageContext';
+
+export const ResetButton = () => {
+  const router = useRouter();
+  const [, dispatch] = useDeploymentPageContext();
+
+  function reset() {
+    localStorage.removeItem('rollupData');
+    localStorage.removeItem('l3Config');
+
+    dispatch({ type: 'reset' });
+
+    router.push('/deployment?step=1');
+  }
+
```

Comment:
If you click on restart and start over, the owner field wouldn’t default to the connected wallet address, instead it'll be empty. not sure how to resolve it